### DISCUSSION TITLE
Can location name be added to display?

### DISCUSSION THREAD

**Discussion by mrtootoo** [breezy-weather/breezy-weather#437] [Ideas]
I like using the widget.  But I can’t tell what location is being used for weather data.  All the widget shows is the words “Current location” instead of the actual location name.

**Comment by papjul** [comment]
Yes, please add a new issue, using the "Idea" template.
I will use the "current location" icon we have on main screen instead so we can both know the city name + the fact that it’s current location.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
@mrtootoo did u create an issue? If so i cant seem to find it

**Reply by mrtootoo** [reply]
I did today.  #513
